cogs/stats_commands.py

```diff
@@ -3,7 +3,9 @@
 from random import choice, randrange
 
 from utils.messages import embedded_stats
+from utils.messages import embedded_matchday_results
```

Comment:
We could have these in a single line since they are importing from the same module.